File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.py
# Last updated: 8/13/2025, 8:24:11 PM
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        current = 0
        for i in range (len(nums)) :
            if nums[i] != nums[current] :
                current+=1
                nums[current] = nums[i]
        
        return current+1
        

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 8/13/2025, 8:24:27 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        int j = 0;

        while(j<nums.length){
            if(nums[i]==nums[j]){
                j++;
            }
            else{
                i++;
                int temp = nums[i];
                nums[i] = nums[j];
                nums[j] = temp;
                j++;
            }
        }
        return i+1;
    }
}